apollo-compiler/src/test/graphql/com/example/fragment_in_fragment/type/CustomScalars.kt

```diff
@@ -0,0 +1,8 @@
+// AUTO-GENERATED FILE. DO NOT MODIFY.
+//
+// This class was automatically generated by Apollo GraphQL plugin from the GraphQL queries it found.
+// It should not be modified by hand.
+//
+package com.example.fragment_in_fragment.type
+
+object CustomScalars
```

Comment:
Should we skip empty object generation?

Reply:
I feel like consistency is a better dev experience and adheres to the principle of least astonishment (see https://github.com/apollographql/apollo-android/issues/2721 for an exemple)
No strong feeling there though. If generating an empty object it causes any problem, we should remove it.

Reply:
I think it will look odd for the user to see empty class generated and visible for import.

Reply:
Also I have a suggestion to generate additional KDoc that briefly explains the purpose of this generated class, what do you think?

Reply:
Yup, +1000 for KDoc, I'll add that. And remove the generated class if empty.

Reply:
Kdoc and skipping empty classes are added